app/views/V2App/app.scala.html

```diff
@@ -52,17 +53,19 @@
             }
             window.$RefreshSig$ = () => (type) => type
             window.__vite_plugin_react_preamble_installed__ = true
-        </script>
+		</script>
         <script type="module" src="http://localhost:5173/assets/fronts-client-v2/@@vite/client"></script>
         <script type="module" src="http://localhost:5173/assets/fronts-client-v2/src/index.tsx"></script>
     } else {
-        <link rel="stylesheet" href="@cssLocation">
+		<link rel="stylesheet" href="@cssLocation">
         <script type="module" src="@jsLocation" type="application/javascript"></script>
     }
 
     @maybePinboardUrl.map { pinboardUrl =>
         <script async defer src="@pinboardUrl"></script>
     }
+
+	@templates.pixel_tracking(maybeTelemetryUrl)
```

Comment:
I think we should be able to remove this component as the tracking will be fired by the client script, causing duplicate visit events.

Reply:
A great spot!